src/reactions/Reaction.js

```diff
@@ -26,24 +28,79 @@ const styles = StyleSheet.create({
   frameNotVoted: {
     borderColor: '#dedede', // eslint-disable-line
   },
-  countCommon: {
+  placeHolderCount: {
     marginLeft: 4,
+    color: 'transparent'
   },
   countVoted: {
     color: BRAND_COLOR,
   },
   countNotVoted: {
     color: 'gray',
   },
+  roller: {
```

Comment:
Use the word 'spinner'. Spin is to rotate around an axis. Roll is similar but there is a movement in some direction, caused by spinning (wheels etc).